apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.module.ts

```diff
@@ -22,6 +23,7 @@ import { AssetProfileDialog } from './asset-profile-dialog.component';
     GfValueModule,
     MatButtonModule,
     MatDialogModule,
+    MatCheckboxModule,
```

Comment:
Can you move up to keep the alphabetical order?

Reply:
Resolved in latest commit.